src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const values = Object.values(robot);
+
+  if (values.length !== new Set(values).size) {
+    return null;
+  }
+
+  const resultObj = {};
+  const params = Object.entries(robot);
```

Comment:
Try to simplify solution Object.values, Set, entries are redundant here. All what you need is just a  one for...in loop